hydra-node/src/Hydra/Options.hs

```diff
@@ -478,6 +478,19 @@ hydraNodeCommand =
       (decodeUtf8 $ encodePretty Contract.scriptInfo)
       (long "script-info" <> help "Dump script info as JSON")
 
+newtype CannotStartHydraNode = CannotStartHydraNode Text deriving (Eq, Show)
+instance Exception CannotStartHydraNode
+
+-- | Validate cmd line arguments for hydra-node
+-- and check if they make sense before actually running the node.
+validateRunOptions :: MonadThrow m => RunOptions -> m ()
+validateRunOptions RunOptions{peers, hydraVerificationKeys} = do
```

Comment:
This one should probably validate that the number of peers matches with the number of loaded cardano/hydra keys as well?